riff-raff.yaml

```diff
@@ -0,0 +1,18 @@
+stacks: [playground]
```

Comment:
What does 'playground' mean here?

Reply:
it's the stack, janus aws account, we need to make a decision of where this will live